scripts/zones/Windurst_Walls/Zone.lua

```diff
@@ -18,6 +18,13 @@ end;
 
 function onZoneIn(player,prevZone)
     local cs = -1;
+
+    if ENABLE_ROV and player:getCurrentMission(ROV) == dsp.mission.id.rov.RHAPSODIES_OF_VANADIEL and player:getMainLvl()>=3 then
```

Comment:
0 evaluates to tru in lua. ENABLE_ROV will always be true in this case. XD

Reply:
gdi bot

Reply:
I was wondering why this wasn't working... 🤡